pkg/client/src/app/common/KeycloakProvider.tsx

```diff
@@ -14,6 +15,17 @@ interface IKeycloakProviderProps {
 export const KeycloakProvider: React.FunctionComponent<
   IKeycloakProviderProps
 > = ({ children }) => {
+  const checkAuthCookie = () => {
+    let currentCookie = getCookie("keycloak_cookie");
+    if (currentCookie !== "" && currentCookie !== null) {
```

Comment:
You can simplify this `if`-`else` since the `if` case is empty here, and just use
```ts
if (currentCookie === "" || currentCookie === null) {
  currentCookie = keycloak?.token || "";
  // ...
```
Or honestly since empty-string and null are both falsy this could just be:
```ts
if (!currentCookie) {
  currentCookie = keycloak?.token || "";
  // ...
```